from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_Restaurante(object):
    def setupUi(self, Restaurante):
        Restaurante.setObjectName("Restaurante")
        Restaurante.resize(800, 600)
        Restaurante.setStyleSheet("background-color: rgb(255, 255, 255);")
        self.centralwidget = QtWidgets.QWidget(Restaurante)
        self.centralwidget.setObjectName("centralwidget")
        self.pushButton = QtWidgets.QPushButton(self.centralwidget)
        self.pushButton.setGeometry(QtCore.QRect(90, 310, 151, 41))
        self.pushButton.setStyleSheet("background-color: rgb(0, 170, 0);")
        self.pushButton.setObjectName("pushButton")
        self.pushButton_2 = QtWidgets.QPushButton(self.centralwidget)
        self.pushButton_2.setGeometry(QtCore.QRect(90, 360, 151, 41))
        self.pushButton_2.setStyleSheet("background-color: rgb(0, 85, 255);")
        self.pushButton_2.setObjectName("pushButton_2")
        self.pushButton_3 = QtWidgets.QPushButton(self.centralwidget)
        self.pushButton_3.setGeometry(QtCore.QRect(90, 410, 151, 41))
        self.pushButton_3.setStyleSheet("background-color: rgb(255, 0, 0);")
        self.pushButton_3.setObjectName("pushButton_3")
        self.label = QtWidgets.QLabel(self.centralwidget)
        self.label.setGeometry(QtCore.QRect(330, 0, 151, 91))
        self.label.setText("")
        self.label.setPixmap(QtGui.QPixmap("image-uFlK5B6rP3-transformed-removebg-preview.png"))
        self.label.setScaledContents(True)
        self.label.setObjectName("label")
        self.label_2 = QtWidgets.QLabel(self.centralwidget)
        self.label_2.setGeometry(QtCore.QRect(50, 100, 261, 171))
        self.label_2.setText("")
        self.label_2.setPixmap(QtGui.QPixmap("../../Downloads/istockphoto-1131230925-612x612.jpg"))
        self.label_2.setScaledContents(True)
        self.label_2.setObjectName("label_2")
        self.lineEdit = QtWidgets.QLineEdit(self.centralwidget)
        self.lineEdit.setGeometry(QtCore.QRect(450, 130, 191, 31))
        self.lineEdit.setObjectName("lineEdit")
        self.lineEdit_2 = QtWidgets.QLineEdit(self.centralwidget)
        self.lineEdit_2.setGeometry(QtCore.QRect(380, 200, 81, 31))
        self.lineEdit_2.setObjectName("lineEdit_2")
        self.lineEdit_3 = QtWidgets.QLineEdit(self.centralwidget)
        self.lineEdit_3.setGeometry(QtCore.QRect(380, 250, 111, 31))
        self.lineEdit_3.setObjectName("lineEdit_3")
        self.lineEdit_4 = QtWidgets.QLineEdit(self.centralwidget)
        self.lineEdit_4.setGeometry(QtCore.QRect(380, 300, 141, 31))
        self.lineEdit_4.setObjectName("lineEdit_4")
        self.lineEdit_5 = QtWidgets.QLineEdit(self.centralwidget)
        self.lineEdit_5.setGeometry(QtCore.QRect(380, 350, 151, 31))
        self.lineEdit_5.setObjectName("lineEdit_5")
        self.lineEdit_6 = QtWidgets.QLineEdit(self.centralwidget)
        self.lineEdit_6.setGeometry(QtCore.QRect(640, 270, 81, 31))
        self.lineEdit_6.setObjectName("lineEdit_6")
        self.lineEdit_7 = QtWidgets.QLineEdit(self.centralwidget)
        self.lineEdit_7.setGeometry(QtCore.QRect(640, 190, 81, 31))
        self.lineEdit_7.setObjectName("lineEdit_7")
        self.lineEdit_8 = QtWidgets.QLineEdit(self.centralwidget)
        self.lineEdit_8.setGeometry(QtCore.QRect(640, 230, 81, 31))
        self.lineEdit_8.setObjectName("lineEdit_8")
        self.pushButton_4 = QtWidgets.QPushButton(self.centralwidget)
        self.pushButton_4.setGeometry(QtCore.QRect(550, 10, 75, 23))
        self.pushButton_4.setObjectName("pushButton_4")
        self.pushButton_5 = QtWidgets.QPushButton(self.centralwidget)
        self.pushButton_5.setGeometry(QtCore.QRect(650, 10, 75, 23))
        self.pushButton_5.setObjectName("pushButton_5")
        Restaurante.setCentralWidget(self.centralwidget)
        self.menubar = QtWidgets.QMenuBar(Restaurante)
        self.menubar.setGeometry(QtCore.QRect(0, 0, 800, 21))
        self.menubar.setObjectName("menubar")
        Restaurante.setMenuBar(self.menubar)
        self.statusbar = QtWidgets.QStatusBar(Restaurante)
        self.statusbar.setObjectName("statusbar")
        Restaurante.setStatusBar(self.statusbar)

        self.retranslateUi(Restaurante)
        QtCore.QMetaObject.connectSlotsByName(Restaurante)

    def retranslateUi(self, Restaurante):
        _translate = QtCore.QCoreApplication.translate
        Restaurante.setWindowTitle(_translate("Restaurante", "MainWindow"))
        self.pushButton.setText(_translate("Restaurante", "ACTIVAR"))
        self.pushButton_2.setText(_translate("Restaurante", "EDITAR"))
        self.pushButton_3.setText(_translate("Restaurante", "DESACTIVAR"))
        self.lineEdit.setText(_translate("Restaurante", "NOMBRE RESTAURANTE X"))
        self.lineEdit_2.setText(_translate("Restaurante", "Usuarios : X"))
        self.lineEdit_3.setText(_translate("Restaurante", "Pedidos activos : X"))
        self.lineEdit_4.setText(_translate("Restaurante", "Ventas por mes : X"))
        self.lineEdit_5.setText(_translate("Restaurante", "Dinero generado al mes : X"))
        self.lineEdit_6.setText(_translate("Restaurante", "Mes"))
        self.lineEdit_7.setText(_translate("Restaurante", "Día"))
        self.lineEdit_8.setText(_translate("Restaurante", "Semana"))
        self.pushButton_4.setText(_translate("Restaurante", "Restaurante"))
        self.pushButton_5.setText(_translate("Restaurante", "Facturación"))
